### PR TITLE
Amend build script to ensure pre-commit hooks pass

### PR DESCRIPTION
**Pull request by JoeChapman**
- git ignore eslintignore file
- remove istanbul coverage tool. No tests means this always fails, breaking commits

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+deploy
+public
+secrets.env
+node_modules
+apps/**/translations/**/default.json
+LICENSE
+README.md
+CONTRIBUTING.md
```
Comment (joefitter) [inline .eslintignore]: should this file have been committed?

Comment (JoeChapman) [inline .eslintignore]: no it should not. one sec

**Comment by joefitter** [comment]
LGTM

**Comment by JoeChapman** [comment]
Can I get a thumb @gxxm @easternbloc ??????????????????

**Pull request merged**